wasmbinding/wasm.go

```diff
@@ -160,8 +166,9 @@ func RegisterCustomPlugins(
 	ek *epochingkeeper.Keeper,
 	ck *checkpointingkeeper.Keeper,
 	lcKeeper *lcKeeper.Keeper,
+	zcKeeper *zckeeper.Keeper,
 ) []wasmkeeper.Option {
-	wasmQueryPlugin := NewQueryPlugin(ek, ck, lcKeeper)
+	wasmQueryPlugin := NewQueryPlugin(ek, ck, lcKeeper, zcKeeper)
```

Comment:
if we had custom msg handler for wasm would also eliminate the possibility to people bypass the epoch staking? @maurolacy

Reply:
Not sure. I would say, no.